Create ui-c8y-1023-78-7-fix-placeholder-in-c-8-y-select-and-add-validation-example.md (#4681)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-78-7-fix-placeholder-in-c-8-y-select-and-add-validation-example.md

---
date: ""

> title: Fix placeholder in c8y-select component and add validation example to Codex

Reviewer: ```suggestion
title: Fixed placeholder in c8y-select component
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66503
version: 1023.78.7
---

> Fixes an issue where the c8y-select placeholder did not display in some cases when no value was selected, and adds a validation example to the documentation.

Reviewer: ```suggestion
Previously, the `c8y-select` placeholder did not display in some cases when no value was selected. This issue has been fixed. Moreover, a validation example has been added to the Codex documentation.
```

> Previously, the `c8y-select` placeholder did not display in some cases when no value was selected. This issue has been fixed. Moreover, a validation example has been added to the Codex documentation.

Reviewer: @carlosceia Can you please provide the link to the documentation?

Reviewer 2: ```suggestion
Previously, the `c8y-select` placeholder did not display in some cases when no value was selected. This issue has been fixed. Moreover, a validation example has been added to the [Codex documentation](https://cumulocity.com/codex/components/forms/select/overview).
```

---
date: ""

> title: fix placeholder in c8y-select and add validation example to codex (#11799) [GRAFT][release/cd] (#11929)

Reviewer 2: ```suggestion
title: Fix placeholder in c8y-select component and add validation example to Codex
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66503
version: 1023.78.7
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-66503] fix placeholder in c8y-select and add
validation example to codex
(#11799)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/11799)

<!--- Backport version: 9.5.1 -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-66503]:
https://cumulocity.atlassian.net/browse/MTM-66503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
Fixes an issue where the c8y-select placeholder did not display in some cases when no value was selected, and adds a validation example to the documentation.
```
